### ISSUE TITLE
[1.16] Lots of problems with Create.

### ISSUE THREAD

**Issue by XionioXMaster** [Loli-Server/LoliServer1.16#93]
**Minecraft Version(我的世界版本) :** 1.16.5

**LoliServer buildVersion(构建版本) :** Provide LoliServer  buildVersion: "LoliServer-1.16.5-38-server.jar"

**Operating System(操作系统) :** Unbuntu 20

**Concerned mod / plugin(有关的插件/Mod) :** Create v0.3.2b

**Logs :** Nope

**Steps to Reproduce(复现步骤) :**
Bug # 1: the harvested crop completely disappears from the automatic farm if you use a regular box. The farm consists of a mechanical rotator, a lane chassis and a mechanical harvester. Video # 1: https://drive.google.com/file/d/1NhPWvhJHvW5MiGWdBMLs-_ZX9Qu8nxXn/view
(if you use an adjustable box, then the crop is saved in it (can be seen in the second vidos))
Bug # 2: the crop from the box is not transferred to the second box using the portable storage interface. Video # 2: https://drive.google.com/file/d/1g9A2TIK9NVYrVtdmSfrejfVjZ0gKA9Wo/view
Bug H3 The standalone activator does not place / breaks blocks
https://user-images.githubusercontent.com/86433158/127630426-e6b6ade1-96ac-466d-80fe-171727451013.mp4

I ran all the tests above on a completely clean Bridge, excluding Create itself.

Basically, as I understand it, the bugs are related to the repository, thanks for your work.
P.S these problems will come out in the future in other mods that use similar mechanics.

**Comment by EvilToasterDBU** [comment]
1 trable - use vanilla barrel 
2 trable - check private, if you use private - deployer requires permission as a fake player

**Comment by XionioXMaster** [comment]
> 1 trable - use vanilla barrel
> 3 trable - check private, if you use private - deployer requires permission as a fake player

Before creating a theme here, I checked everything exclusively with this mod, without mods for private and the like.

**Comment by XionioXMaster** [comment]
Bug H3 don't work, use WorldGuard

**Comment by DonorKrovi** [comment]
这个问题非常难解决。
